src/test/recovery/t/002_columnar_crash_recovery.pl

```diff
@@ -0,0 +1,44 @@
+# Minimal test testing streaming replication
+use strict;
+use warnings;
+use PostgresNode;
+use TestLib;
+use Test::More tests => 2;
+
+# Initialize single node
+my $node_one = get_new_node('node_one');
+$node_one->init();
+$node_one->start;
+
+# initialize the citus extension
+$node_one->safe_psql('postgres', "CREATE EXTENSION citus;");
+
+# create columnar table and insert simple data to verify the data survives a crash
+$node_one->safe_psql('postgres', "
+BEGIN;
+CREATE TABLE t1 (a int, b text) USING columnar;
+INSERT INTO t1 SELECT a, 'hello world ' || a FROM generate_series(1,1002) AS a;
+COMMIT;
+");
```

Comment:
I'd suggest having one committed transaction and one open transaction. Maybe a prepared xact as well?